src/modules/modules.py

```diff
@@ -167,7 +167,11 @@ def forward(self, sent, task, reset=True):
         sent_mask = util.get_text_field_mask(sent).float()
         sent_lstm_mask = sent_mask if self._mask_lstms else None
         if sent_embs is not None:
-            sent_enc = self._phrase_layer(sent_embs, sent_lstm_mask)
+            if isinstance(self._phrase_layer, ONLSTMStack):
+                sent_enc, _ = self._phrase_layer(torch.transpose(sent["words"],0,1), sent_lstm_mask)
```

Comment:
Can this logic just be moved inside the `ONLSTMStack` class so there's no special case here?

Alternatively, write a wrapper class similar to `src/bert/utils.py`.

Reply:
?

Reply:
Added a separate task type parallel to lm.py for LM parsing to handle this in a neat manner.

Reply:
Trying to fix this, but here is the issue I am running into: the ONLSTM module runs embedded dropout separately so it needs the raw words rather than just the sentence embeddings.